### ISSUE TITLE
Issue: Cannot expose a BLOB field in power automate  

### ISSUE THREAD

**Issue by IsElon** [hougaard/Simple-Object-Designer#198]
Hi there, 
I have been trying to expose a BLOB field called "Work Description" in the power automate, but so far I haven't had any luck. 

Here's what I did: I added a page and picked various fields that I want such as Shipment Date, Work Description, etc., using web services and published it, all the fields came through to the power automate, except for this "Work Description". 

So, my question is, do we have a solution to bring in this BLOB field ?

**Comment by hougaard** [comment]
Right now, we don't have a solution. Power Automate doesn't support blobs the way they're exposed be default from BC. I have a plan, but no ETA on that part.

**Comment by IsElon** [comment]
> Right now, we don't have a solution. Power Automate doesn't support blobs the way they're exposed be default from BC. I have a plan, but no ETA on that part.

Appreciate for the response. Also, I remember somehow I found a table in web services that has the Work description field, and I gave it a String value in power automate, and then it got populated into Wiise. Does it mean there might be a type conversion behind this Work Description field?

**Comment by hougaard** [comment]
You can try, maybe hit that API in Postman to see how the data is shown.

**Comment by teganZT** [comment]
I am also experiencing this issue with the Work Description field in the Sales Header table.

**Comment by hougaard** [comment]
What would be the right appoach, some magic behind the scene to treat that field as a plain text field?

**Comment by teganZT** [comment]
Yes, for my use case I need to be able to read/update plain text in the field using PowerAutomate.

**Comment by wadepierce** [comment]
> Yes, for my use case I need to be able to read/update plain text in the field using PowerAutomate.

Ditto!
